crates/cli/src/subcommands/generate/mod.rs

```diff
@@ -372,16 +377,22 @@ impl GenItem {
 }
 
 pub fn extract_descriptions(wasm_file: &Path) -> anyhow::Result<RawModuleDef> {
-    let engine = wasmtime::Engine::default();
-    let t = std::time::Instant::now();
-    let module = wasmtime::Module::from_file(&engine, wasm_file)?;
-    println!("compilation took {:?}", t.elapsed());
```

Comment:
did we remove this summary message entirely, or does the new spinner do the same?

Reply:
I guess either way users could just run things through `time` if they're trying to do a before-and-after comparison of something.